Packs/Slack/Integrations/SlackV3/SlackV3.py

```diff
@@ -1188,12 +1215,14 @@ async def translate_create(message: str, user_name: str, user_email: str, demist
             incident_type = incident_type or INCIDENT_TYPE
             if incident_type:
                 incident['type'] = incident_type
-
+            incidents = add_req_data_to_incidents([incident], request_fields)
             created_incident = await create_incidents([incident], user_name, user_email, user_demisto_id)
```

Comment:
It's correct, but note that it's not completely trivial to understand that in the call to `create_incidents`, the `[incident]` argument there contains the rawJSON. I'd assign `incidents = [incident]` before, and then pass it to both methods.